src/library/Button/index.d.ts

```diff
@@ -1 +1,29 @@
-declare module '@targetx/mineral-ui/Button';
+import { ElementType, ReactElement, ReactNode, SyntheticEvent } from 'react';
+
+type Size = 'small' | 'medium' | 'large' | 'jumbo';
+type Variant = 'danger' | 'grayscale' | 'success' | 'warning';
+
+export declare namespace Button {
+  export interface Props {
+    as?: ElementType;
+    children?: ReactNode;
+    circular?: boolean;
+    css?: { [key: string]: any };
+    disabled?: boolean;
+    fullWidth?: boolean;
+    href?: string;
+    iconEnd?: ReactElement;
+    iconStart?: ReactElement;
+    justifyContent?: string;
+    minimal?: boolean;
+    onClick?: (event: SyntheticEvent) => void;
```

Comment:
@tx-nestor @tx-steven 
This change on 19 is breaking in multiple projects. TX-Print breaks in several components and If I install this in `td-core-web-ui`, that project also breaks. I don't see any open pr's to address this in core-web-ui. How would we refactor the projects to conform to this type? I'm unable to adapt my usage of event in tx-print to this type on line 19.

Reply:
In td-core-web-ui

![image](https://user-images.githubusercontent.com/50591254/65067369-3101a500-d93b-11e9-8372-e06bae0b4098.png)

Reply:
In tx-print:
![image](https://user-images.githubusercontent.com/50591254/65067468-702ff600-d93b-11e9-826f-46face344e1d.png)


If I change `HTMLButtonElement` to `Eleement`, I then lose access to the `value` property on event.target